WPMedia/fusion-news-theme-blocks
# Pen 1527 ad callback label

## Description
- toggle advertisement label only after ad callback from google
- clean up some code left over from PEN-1508 (remove display customfield)

## Jira Ticket
- [PEN-1527](https://arcpublishing.atlassian.net/browse/PEN-1527)

## Acceptance Criteria
- only show advertisement label while customfields is checked and after google ad callback

## Test Steps
- pull this branch locally and run `rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..`
- in `Fusion-News-Theme` repo, run `npx fusion start -f -l`
- once in Fusion started, create new page for ad testing with right rail layout
- on the new page, add multiple Google ad blocks on Main with 'leaderboard' ad type , and 'right rail' ad type
<img width="279" alt="Screen Shot 2020-12-14 at 10 04 39 PM" src="https://user-images.githubusercontent.com/4411687/102177509-8c7d5400-3e58-11eb-8d7e-17bcb6140138.png">
- once finished, 'stage and published' the page and open the publish page in desktop view.
- the advertisement labels should show on all ads (assume the checkbox in page builder for display label is checked)
- if you are in chrome, toggle to mobile view, and inspect the right rail ads should no longer showing label

## Effect Of Changes
### Before
Advertisement labels show on right rail ads type in Google Ad block in mobile view

### After
Advertisement labels NO LONGER show on right rail ads type in Google Ad block in mobile view


## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- some ad unit may not be set up

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
